src/lib/customTraceType.js

```diff
@@ -45,29 +43,10 @@ export function traceTypeToPlotlyInitFigure(traceType, gl = '') {
       return {type: 'scatter' + gl, mode: 'lines', stackgroup: 1};
     case 'scatterpolar':
       return {type: 'scatterpolar' + gl};
-    case 'ohlc':
-      return {
-        type: 'ohlc',
-        decreasing: {line: {color: COLORS.middleGray}},
-        increasing: {line: {color: COLORS.blueTeal}},
-      };
     case 'waterfall':
       return {
         type: 'waterfall',
-        decreasing: {line: {color: COLORS.middleGray}},
-        increasing: {line: {color: COLORS.blueTeal}},
-      };
-    case 'candlestick':
-      return {
-        type: 'candlestick',
-        decreasing: {
-          line: {color: COLORS.middleGray},
-          fillcolor: 'rgba(127, 127, 127, 0.5)',
-        },
-        increasing: {
-          line: {color: COLORS.blueTeal},
-          fillcolor: 'rgba(23, 190, 207, 0.5)',
-        },
+        orientation: 'v',
```

Comment:
well, I was asked for those colors previously.. but if we don't need them anymore ok..

Reply:
yeah I don't think it makes sense personally.